stacks-signer/src/ping/runloop.rs

```diff
@@ -0,0 +1,130 @@
+use stacks_common::{debug, trace};
```

Comment:
Why is there a wholly separate runloop?  Can't the ping/pong just be part of the main runloop?

Reply:
Acknowledged, I'll shift efforts into a unified RunLoop.
On an earlier attempt, I had a single run loop. I noticed high variance (I had seen 200ms vs 15s frequently). My unbacked PoV is that there are many points in the RunLoop where it could be stalled. Waiting on commands, retry loops with exponential back-off requests to the stacks-node, expensive crypto computation, etc. And it could add a lot of variance. But what I had implemented wasn't RTT. It was one-way latency. On my second attempt, I right away scraped the single Runloop in favor of a dedicated one.